spec/factories/events.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :event do
+    title 'event title'
+    body 'event content'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.